src/components/Form/SearchInput.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import T from 'prop-types'
+import { randomId, classNames } from '../../utils/'
+
+const SearchInput = ({ id, className, small, placeholder = 'Search...', ...rest }) => (
```

Comment:
We should be using the Form.TextInput component here.

Reply:
done